Erreur inconnue (veuillez faire un rapport de bug) : <type 'exceptions.AttributeError'>\n
Sa valeur est : AttributeError("type object 'plm.universe.Direction' has no attribute 'South'",)
  File "<script>", line 13, in <module>
